packages/wasm/src/services/credential/pex-helpers.js

```diff
@@ -49,23 +60,37 @@ function getAttributeName({field, selectedCredentials, index}) {
   return attributeName;
 }
 
-export function pexToBounds(
-  pexRequest,
-  selectedCredentials = [],
-  isRawBoundies = false,
```

Comment:
not sure what this was used for, but the callers to pexToBounds dont use it so assumed it can be safely removed